utils/roles/control_plane_cleanup/vars/main.yml

```diff
@@ -105,10 +106,9 @@ vault_file_perm: '0644'
 local_repo_config_file: "{{ role_path }}/../../../input/local_repo_config.yml"
 local_repo_config_syntax_fail_msg: "Failed. Syntax errors present in local_repo_config.yml. Fix errors and re-run playbook again. Common syntax Errors:
 indentation errors, improper quotes, improper space or tab, missing colon, missing comma etc. "
-docker_list_path: "/etc/apt/sources.list.d/docker.list"
 
 # Usage: clean_nfs_server.yml
 storage_config_path: "{{ role_path }}/../../../input/storage_config.yml"
 provision_metadata_path: "/opt/omnia/.data/metadata.yml"
 exports_file_path: /etc/exports
-cleanup_nfs_server_fail_msg: "Failed to cleanup nfs server. Check syntax error in input/storage_config.yml and re-run playbook again."
+cleanup_nfs_server_fail_msg: "Failed to cleanup nfs server. Check syntax error in input/storage_config.yml and re-run playbook again."
```

Comment:
Add return line at the EOF

Reply:
Addressed changes in internal PR. It will be available in next set of PRs